tests/dialects/test_bigquery.py

```diff
@@ -2499,6 +2499,14 @@ def test_unix_seconds(self):
         for dialect in ("bigquery", "spark", "databricks"):
             parse_one("UNIX_SECONDS(col)", dialect=dialect).assert_is(exp.UnixSeconds)
 
+    def test_unix_micros(self):
+        self.validate_all(
+            "SELECT UNIX_MICROS('2008-12-25 15:30:00+00')",
+            write={
+                "duckdb": "SELECT EPOCH_US('2008-12-25 15:30:00+00')",
```

Comment:
We should generally ensure roundtrip is unaffected when adding tests like this:

```suggestion
                "bigquery": "SELECT UNIX_MICROS('2008-12-25 15:30:00+00')",
                "duckdb": "SELECT EPOCH_US('2008-12-25 15:30:00+00')",
```